src/mlp_regressor.py

```diff
@@ -0,0 +1,82 @@
+"""
+this module is for mlp regression
+"""
+import os
+
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.neural_network import MLPRegressor
+from sklearn.model_selection import train_test_split, GridSearchCV
+from sklearn.preprocessing import StandardScaler
+
+
+class MLPRepressor:    # pylint: disable=R0902
+    """
+    this class is for the mlp classifier
+    """
+
+    def __init__(self, data_x, data_y, param_grid):
+        self.data_x = data_x
+        self.data_y = data_y
```

Comment:
- You never use these lines: `self.data_x = data_x, self.data_y = data_y`.
- They can be deleted.